### ISSUE TITLE
Transaction status ERROR

### ISSUE THREAD

**Issue by dietzam** [iExecBlockchainComputing/public-pool-scripts#42]

**Comment by dietzam** [comment]
0x3240711194f05ae4070f15a6cb6092f5931b4575

I had 2 tasks allocated, but both of them have status of ERROR. Unfortunately they no longer are seen in task history.

**Comment by branciard** [comment]
I see you contribution are valid. you can see the workeroder id in the data of your transaction : 0x38b5c38c2e35048b36648806995d20021ed08cac 
But the workerorder as an issue :  

```


iexec work show 0x38b5c38c2e35048b36648806995d20021ed08cac --chain mainnet
ℹ iExec SDK update available 2.2.39 →  2.3.1, Run "npm -g i iexec" to update

ℹ using chain [mainnet]
✔ work 0x38b5c38c2e35048b36648806995d20021ed08cac status is ACTIVE, details:
m_workerpool:          0x0ed5e70874487bf21ee65343fb6c5fa13382cdb6
m_params:              {"cmdline":"monte-carlo normal"}
m_requester:           0x8ec966ff9752c5de46105155ef1e9a3a38542358
m_app:                 0xcb0bba8cf239e167af92a51c4212adbf9b479f6a
m_dataset:             0x0000000000000000000000000000000000000000
m_emitcost:            0
m_uri:
m_stdout:
m_resultCallbackProof: 0x0000000000000000000000000000000000000000000000000000000000000000
m_iexecHubAddress:     0x0d5ef019ca4c5cc413ee892ced89d7107c5f424d
m_callback:            0x0000000000000000000000000000000000000000
m_status:              1
m_marketorderIdx:      317
m_stderr:
m_beneficiary:         0x0000000000000000000000000000000000000000
m_statusName:          ACTIVE
```


I will manual reward you after the workerdrop.

**Comment by branciard** [comment]
23 RLC manual reward done for worker 0x3240711194f05ae4070f15a6cb6092f5931b4575 see tx : https://etherscan.io/tx/0x6335636c76e750950b05ea7071b70a5cf6fec72ceb55c3fa000c6694202f3919
